README.md

```diff
@@ -54,13 +54,13 @@ code in this repository
 
 [./components/](components) contains the source for each component, and its
   FFI bindings.
-* See [./components/logins/](components/logins) for an example, where you can
+* See [./components/push/](components/places) for an example, where you can
     find:
-  * The shared [rust code](components/logins/src).
-  * The mapping into a [C FFI](components/logins/ffi).
-  * The [Kotlin bindings](components/logins/android) for use by Android
+  * The shared [rust code](components/places/src).
+  * The mapping into a [C FFI](components/places/ffi).
```

Comment:
It might also be worth adding a disclaimer here - something like "(although note that we are moving away from hand-written FFI bindings in favour of [uniffi](https://github.com/mozilla/uniffi-rs/))"